homeassistant/components/geniushub/__init__.py

```diff
@@ -73,6 +73,17 @@
 
 SVC_SET_ZONE_MODE = "set_zone_mode"
 SVC_SET_ZONE_OVERRIDE = "set_zone_override"
+SVC_SET_SWITCH_OVERRIDE = "set_switch_override"
+
+SET_SWITCH_OVERRIDE_SCHEMA = vol.Schema(
+    {
+        vol.Required(ATTR_ENTITY_ID): cv.entity_id,
+        vol.Required(ATTR_DURATION): vol.All(
```

Comment:
Make this `vol.Optional` - default duration 1 hour

Reply:
I don't think making this parameter optional makes sense.
What is the 1 hour based on?

Reply:
The hour is based on the default set by the hub and has the same effect as `switch.turn_on`.